ceph.spec.in

```diff
@@ -57,7 +57,11 @@
 %global _remote_tarball_prefix https://download.ceph.com/tarballs/
 %endif
 %if 0%{?suse_version}
+%ifarch s390x
```

Comment:
Did you see that it was also disabled for aarch64 in a followup -- https://github.com/ceph/ceph/pull/47544.  It looks like SUSE doesn't need that but I wanted to double check.

Reply:
Good catch :-) but it's OK, SUSE does have an aarch64 build of pmdk.